src/wazuh_modules/vulnerability_scanner/tests/unit/factoryOrchestrator_test.cpp

```diff
@@ -28,6 +28,9 @@ typedef enum HANDLERS
     INVENTORY_SYNC,
     SEND_REPORT,
     RESULT_INDEXER,
+    CLEAN_UP_DATA,
+    FETCH_FROM_GLOBAL_DB,
+    QUERY_ALL_PACKAGES,
```

Comment:
Improve (design): This whole enum should be defined on the hpp file so we can use it on the other tests

Reply:
ditto